### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by nclsHart**
Travis will now use a fixed version of phpunit instead of the default one.

### CODE REVIEW

**Comment by hkdobrev** [comment]
I'm personally for having PHPUnit as a composer dependency, but I'll let @ivank chime in on that. Now when we have Composer packages cached in Travis CI this would not slow the build a lot.

Could you elaborate more on why you have moved the `swiftmailer/swiftmailer` dependency to the `require` section? Thanks!

**Comment by nclsHart** [comment]
Hi,

> Could you elaborate more on why you have moved the swiftmailer/swiftmailer dependency to the require section? Thanks!

With the current configuration, when running an install in a production environment (using `--no-dev`), composer will ignore your requirements. And if another package depends on another version of swiftmailer which is not compatible with yours, composer will say everything is alright. 

For example, if you have a project with the following `require` section : 

```json
"require": {
       "swiftmailer/swiftmailer": "^4.1.0",
       "openbuildings/swiftmailer-css-inliner": "^0.3"
}
```
=> there is no conflict and swiftmailer 4.1.8 is installed. And with this PR you'll now have a conflict because of a non resolvable requirements.

File: composer.json [hunk 1]
```diff
@@ -11,10 +11,11 @@
     ],
     "require" : {
         "php": ">=5.4",
+        "swiftmailer/swiftmailer": "^5.1",
         "tijsverkoyen/css-to-inline-styles": "^1.5"
     },
     "require-dev" : {
-        "swiftmailer/swiftmailer": "^5.1"
+        "phpunit/phpunit": "^4.0"
```
Comment (hkdobrev) [inline composer.json]: @nclsHart Do you think we could allow both 4.x and 5.x versions of PHPUnit here? Or just upgrade to the 5.x latest stable version?

Comment (nclsHart) [inline composer.json]: @hkdobrev If unit tests are compatible with 5.x, I don't see any problem to use the latest stable.

If you are ok, I may update the pull request.

Comment (nclsHart) [inline composer.json]: @hkdobrev 

EDIT :  phpunit 5.x requires php >=5.6. So if you want to be compatible with php >=5.4 you need to stay with phpunit 4.x or allow both versions.

**Pull request closed without merging**